src/components/controls/mouse-controls.js

```diff
@@ -0,0 +1,157 @@
+var registerControls = require('../../core/controls').registerControls;
+var THREE = require('../../lib/three');
+var debug = require('../../utils/debug');
+
+var warn = debug('components:mouse-controls:warn');
+
+/**
+ * Mouse-controls component.
+ *
+ * Updates rotation with mousedrag or pointerlock input.
+ */
+module.exports.Component = registerControls('mouse-controls', {
+  schema: {
+    enabled: { default: true },
+    pointerlockEnabled: { default: false },
```

Comment:
Though we'd keep mouse drag as a bare minimum so people can at least preview experiences. Mouse drag is a lot easier to test, more predictable, and Pointer Lock has too many steps (browser modals) if you just want to preview around.